content/en/docs/concepts/configuration/secret.md

```diff
@@ -381,7 +381,7 @@ The following YAML contains an example config for a TLS Secret:
 
 The TLS Secret type is provided only for convenience.
 You can create an `Opaque` type for credentials used for TLS authentication.
-However, using the defined and public Secret type (`kubernetes.io/ssh-auth`)
+However, using the defined and public Secret type (`kubernetes.io/tls`)
```

Comment:
Hello @luoxiaohi .
Do you have any more information on this change? It's not clear that the `kubernetes.io/tls` Secret type is correct.

Reply:
This matches line 364.